caikit_nlp/modules/text_generation/peft_config.py

```diff
@@ -96,12 +96,142 @@ def resolve_base_model(base_model, cls, torch_dtype):
 
 
 def get_peft_config(
-    tuning_type, tuning_config, base_model, cls, torch_dtype, verbalizer
+    tuning_type,
+    tuning_config,
+    base_model,
+    cls=None,
+    torch_dtype=None,
+    verbalizer="{{input}}",
```

Comment:
is there a reason these defaults are now being set here? I.e., the calling module is always going to pass them in positionally and override these defaults, right?